src/docs/content/Introduction.mdx

```diff
@@ -1,8 +1,8 @@
-import { DocCollapsible } from "src/docs/components/InstallationDocs";
+import { CTA } from "src/docs/components/CTA";
 import { Tabs } from "@components/Tabs";
 import { TabsContent, TabsList, TabsTrigger } from "@components/Tabs/Tabs";
 
-# Getting started
+# MailingUI components
```

Comment:
I would name this page `Installation` and check the headers.

This makes sense to me, open to proposals. I know not perfect but we can change once we add more pages.

<img width="1624" alt="image" src="https://github.com/webscopeio/mailingui/assets/77650775/c96b1f44-1653-4f4d-a3f9-a1d1b1187dd3">